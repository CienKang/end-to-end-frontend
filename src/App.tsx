import React from 'react';
import { BrowserRouter, Routes, Route } from 'react-router-dom';
import './App.css';
import { ProtectedRoute } from './components';

import { LoginPage } from './pages';

function App() {
    return (
        <BrowserRouter>
            <Routes>
                <Route path='/login' element={<LoginPage />} />
                <Route path='/register' element={<LoginPage />} />
            </Routes>
        </BrowserRouter>
    );
}

export default App;
