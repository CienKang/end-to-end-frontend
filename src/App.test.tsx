import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

describe('App', () => {
    it('renders without crashing', () => {
        render(<App />);
    });

    it('should match with previous snapshot', () => {
        const { asFragment } = render(<App />);
        expect(asFragment()).toMatchSnapshot();
    });

});