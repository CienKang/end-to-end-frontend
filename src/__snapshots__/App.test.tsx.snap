// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match with previous snapshot 1`] = `<DocumentFragment />`;
